test/fixtures/provisionedConcurrency/serverless.yml

```diff
@@ -22,3 +22,18 @@ functions:
                 - Ref: AWS::Region
                 - Ref: AWS::AccountId
                 - ${self:service.name}-provisioned
+  provisionedKinesis:
```

Comment:
How about using same function for testing both?

That'll probably ensure that we do not prolong integration test time.

It feels a bit quirky, but I think it still tests well setup (or do I miss something?)

`confirmCloudWatchLogs` has `checkIsComplete` option, through which we may ensure that second test waits for right logs to confirm that trigger worked

Reply:
Thanks about the suggestion to use `checkIsComplete` :+1: I've migrated to one function that handles both integration test cases